qiskit/visualization/dag_visualization.py

```diff
@@ -27,6 +29,7 @@
 
 
 @_optionals.HAS_GRAPHVIZ.require_in_call
```

Comment:
Is this graphviz needed now?

Reply:
Yeah, it's still needed because we are subprocessing out to call graphviz internally. We're just doing it directly now instead of via `graphviz_draw()`